src/test/scala/me/jeffmay/util/akka/TestGlobalAkka.scala

```diff
@@ -0,0 +1,40 @@
+package me.jeffmay.util.akka
+
+import java.util.concurrent.ThreadFactory
+
+import akka.actor.{ActorSystem, Scheduler}
+import akka.dispatch.MonitorableThreadFactory
+import org.joda.time.{DateTime, DateTimeZone}
+
+import scala.concurrent.duration.FiniteDuration
+
+object TestGlobalAkka {
+
+  private[akka] lazy val threadFactory: ThreadFactory = MonitorableThreadFactory(
+    "global-akka-scheduler",
+    daemonic = true,
+    contextClassLoader = Option(getClass.getClassLoader)
+  )
+
+  def scheduler: Scheduler = Implicits.scheduler
+
+  def system: ActorSystem = Implicits.system
+
+  def throttler(interval: FiniteDuration): NonBlockingCountingThrottler = {
+    new NonBlockingCountingThrottler(
+      scheduler = scheduler,
+      delay = interval,
+      onStartAction = n => println(s"${DateTime.now(DateTimeZone.UTC)} [GlobalClientLock] Action #$n: Starting..."),
+      onCompleteAction = n => println(s"${DateTime.now(DateTimeZone.UTC)} [GlobalClientLock] Action #$n: Completed."),
+      onStartDelay = n => println(s"${DateTime.now(DateTimeZone.UTC)} [GlobalClientLock] Adding delay of $interval before next action..."),
+      onEndDelay = n => println(s"${DateTime.now(DateTimeZone.UTC)} [GlobalClientLock] Delay finished. Ready for next action.")
+    )
+  }
```

Comment:
FAIL: I removed this `NonBlockingCountingThrottler` because it was a poor man's version of an Actor. This will cause compiler errors.